tests/test_analyze_common.py

```diff
@@ -248,6 +251,19 @@ def testGetInstalledPackageNamesWithRemoveFlag(self):
         self.assertEqual(
             len(common.get_installed_package_names(self.command2)), 0)
 
+    def testGetGitURL(self):
```

Comment:
In the latest commit, i change the test from test_analyze_dockerfile.py to test_analyze_common.py, since testing the url list is more convient than testing the comments and also needs less modifications.

Reply:
Can you post a test result from a Dockerfile here then?

Reply:
I ran a local test on `tests/dockerfiles/pin_add_command_test/pin_add_command_test_dockerfile`
Here is the test code
`def testCreatLockedFile(self):`
     `dfobj = dockerfile.get_dockerfile_obj(self.testfile)`
     `dffile = dockerfile.create_locked_dockerfile(dfobj)`
     `dockerfile.write_locked_dockerfile(dffile, self.testfile + '_lock')`
I have some problems with uploading the locked file, so i just paste the content below(2 lines).
`FROM ubuntu@sha256:04d48df82c938587820d7b6006f5071dbbffceb7ca01d2814f81857c631d44df`
`ADD plain_file /tmp # git project name: tern, HEAD sha: 0f84e2099e085fd085a6e6348b8fa411011ff57f, project url: github.com/ForgetMe17/tern github.com/vmware/tern 
`